pkg/volume/iscsi/iscsi_util.go

```diff
@@ -167,7 +171,11 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 
 	iscsiTransport = extractTransportname(string(out))
 
-	bkpPortal := b.portals
+	bkpPortal := removeDuplicate(b.portals)
+	if len(bkpPortal) == 0 {
```

Comment:
Its perfectly valid that portal can be empty. I may be missing the logic here, can you please fill me in?